static/templates/task-feed/main.html

```diff
@@ -1,106 +1,71 @@
 <div class="_main-content _middle-content">
-  <div layout="row" class="layout-align-start-center _content-nav">
-
-    <span class="_nav-title" style="margin-left: 30px">Task Feed</span>
-
-    <div class="_content-actions">
-
-      <md-button class="_nav-actions" ng-click="">
-        <md-icon md-font-set="material-icons">more_vert</md-icon>
-        <md-tooltip>Actions</md-tooltip>
-      </md-button>
-    </div>
+  <div layout="row" class="layout-align-start-center _task-nav md-whiteframe-z2">
+    <span class="_nav-title _left-inset">Task Feed</span>
   </div>
-  <div class="_main-content-body">
-
-    <md-card class="fx-fade-down fx-cubic" ng-hide="!project.available_tasks"
-             ng-repeat="project in taskfeed.projects" ng-click="taskfeed.showPreview(project)"
-             hover-class="md-whiteframe-z1">
-      <md-card-content>
-        <div layout="row">
-          <span class="inset">
-              <span class="md-title">{{ project.name }}</span>
-          </span>
-        </div>
-        <div layout="row" class="sub-row">
-            <span class="inset">
-                <span>{{ project.age }} by
-                    {{ project.owner.alias }}
-                </span>
-            </span>
-            <span>
-                <span class="inset">
-                    <md-icon md-font-set="material-icons">attach_money</md-icon>
-                    USD {{ project.price }}
-                </span>
-                <span class="inset">
-                    <md-icon md-font-set="material-icons">list</md-icon>
-                    Tasks (available/total): {{ project.available_tasks + '/'+project.total_tasks}}
+  <div class="_feed-list">
+    <div ng-hide="!project.available_tasks" ng-repeat="project in taskfeed.projects" class="md-whiteframe-z2">
```

Comment:
z2 seems a bit too much, we probably have to go for a custom one that matches Chris' design